src/batch_operations.ts

```diff
@@ -0,0 +1,88 @@
+import StreamClient, { APIResponse, UnknownRecord } from './client';
+import utils from './utils';
+
+type BaseFollowRelation = {
+  source: string;
+  target: string;
+};
+
+export type FollowRelation = BaseFollowRelation & {
+  activity_copy_limit?: number;
+};
+
+export type UnfollowRelation = BaseFollowRelation & {
+  keep_history?: boolean;
+};
+
+function addToMany<ActivityType extends UnknownRecord = UnknownRecord>(
+  this: StreamClient,
+  activity: ActivityType,
+  feeds: string[],
+) {
+  /**
```

Comment:
Docs should come before function declaration, not in body. I guess otherwise, ts won't pick up it.

Reply:
https://github.com/GetStream/stream-js/pull/368